app/controllers/katello/api/v2/content_view_filters_controller.rb

```diff
@@ -69,25 +68,6 @@ def destroy
       respond_for_show :resource => @filter
     end
 
-    api :GET, "/content_views/:content_view_id/filters/:id/available_package_groups",
-        N_("Get package groups that are available to be added to the filter")
-    api :GET, "/content_view_filters/:id/available_package_groups",
-        N_("Get package groups that are available to be added to the filter")
```

Comment:
We need to either deprecate this APIs in 2.3 (so we can release their removal in 2.4) or change them such that they call the new code and continue to work from these two endpoints.